### ISSUE TITLE
Sensor Type should return 1024, not 11

### ISSUE THREAD

**Issue by rdpoor** [rdpoor/cp#41]
The Sensor Type is actually returning the incorrect value. Your documentation says it should return 1024, which I agree with, but the actual data returned is x00 x0B, which is 11 decimal. I think I see the confusion because that is bit position 11, but it should be returning the hex value with only bit 11 set. This would be x04 x00 or 1024 decimal. See the excerpt from the original MB spec below.

**Comment by rdpoor** [comment]
Yep, you are correct -- I'm writing the bit position, not the single bit.  But I remember why I did this: In the original 2.4 spec, Sensor Type could not be implemented as specified.  Specifically:

Device          Bit Position      
pH              16 (MSB)
EM802/pH        15 
...
Conductivity    11
...
Unused          1
Unused          0

A bit position of 0 is a value of 0x0001, bit position 1 is 0x0002, etc.  A bit position of 15 is 0x8000, and a bit position of 16 doesn't fit in a 16 bit register.  This begs the question: what's the right value for a pH sensor type?

I believe the correct solution is to throw away the lowest Unused slot and reduce the bit position by one for all the remaining instruments, so the spec would look like this:

Device          Bit Position      
pH              15 (MSB)
EM802/pH        14 
...
Conductivity    10
...
Unused          1
Unused          0

With this in mind, I agree the Sensor Type value for the Conductivity meter is 1024, not 11.

Bottom line:
* This requires a documentation change, the firmware doesn't need to change
* This requires a change to SensorControl
* The firmware doesn't need to change.

**Comment by rdpoor** [comment]
The current design in SensorControl/instrument.py only allows one sensor type at a time -- this needs to be fixed.  But the use of bit positions implies that multiple sensor types may be joined together.

What's the right way to display and input that at the user level?  Some possibilities:

1. We could use decimal numbers, e.g. where '1024' signifies a conductivity probe and `40960` signifies a pH probe (1 << 15) with an ORP probe (1 << 13).
2. We could do the same thing but using hexadecimal: '0x400' for a conductivity probe and '0xa000' for pH with ORP
3. We could use symbolic names, separated by plus signs, like 'Conductivity' and 'pH + ORP' (it's a bit more difficult, but easiest for the user).

Thoughts?

**Comment by rdpoor** [comment]
In conversation:

* Only one Sensor Type will be enabled at a time
* Internal representation is a power of two
* Display / accept symbolic strings
